src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.html

```diff
@@ -0,0 +1,57 @@
+<tm-session-edit-form [formMode]="SessionEditFormMode.EDIT" [(model)]="sessionEditFormModel" (editExistingSessionEvent)="editExistingSessionHandler()"
+                      (deleteExistingSessionEvent)="deleteExistingSessionHandler()"></tm-session-edit-form>
+
+<tm-question-edit-form *ngFor="let questionEditFormModel of questionEditFormModels; let i = index; trackBy: trackQuestionEditFormByFn"
+                       [formMode]="QuestionEditFormMode.EDIT"
+                       [numOfQuestions]="questionEditFormModels.length"
+                       (saveExistingQuestionEvent)="saveExistingQuestionHandler(i)"
+                       (discardExistingQuestionChangesEvent)="discardExistingQuestionHandler(i)"
+                       (duplicateCurrentQuestionEvent)="duplicateCurrentQuestionHandler(i)"
+                       (deleteCurrentQuestionEvent)="deleteExistingQuestionHandler(i)"
+                       [(formModel)]="questionEditFormModels[i]"
+></tm-question-edit-form>
+
+<br/>
+<div class="card" *ngIf="!isAddingQuestionPanelExpanded">
+  <div class="card-body background-color-light-blue text-center">
+    <div ngbDropdown class="d-inline-block">
+      <button class="btn btn-primary" ngbDropdownToggle>Add New Question</button>
+      <div ngbDropdownMenu>
+        <button class="dropdown-item">Choose from template questions</button>
```

Comment:
Is this option necessary?

Reply:
It will trigger a modal.

![image](https://user-images.githubusercontent.com/13977235/51120956-2b563b80-1851-11e9-9d38-bf8fa8901dc2.png)

Looks like I can implement this easily. Will do.